From 5277168d1691ce40fc04fb4adefd89d8eb39c81e Mon Sep 17 00:00:00 2001
From: chrome-release-bot <chrome-release-bot@chromium.org>
Date: Sat, 27 Jun 2020 18:07:45 +0000
Subject: [PATCH 11/12] Incrementing VERSION to 83.0.4103.122

TBR=lakpamarthy@chromium.org

Change-Id: I5b90c234baff6204fb1684a4d99878771a112e38
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2271618
Reviewed-by: Chrome Release Bot (LUCI) <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/4103@{#734}
Cr-Branched-From: 8ad47e8d21f6866e4a37f47d83a860d41debf514-refs/heads/master@{#756066}
---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index ff22ff08a83..ea9ff344413 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=83
 MINOR=0
 BUILD=4103
-PATCH=121
+PATCH=122
-- 
2.28.0

